src/main/kotlin/no/nav/syfo/behandlendeenhet/kafka/BehandlendeEnhetProducer.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.syfo.behandlendeenhet.kafka
+
+import no.nav.syfo.behandlendeenhet.database.domain.PPerson
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class BehandlendeEnhetProducer(
+    private val kafkaProducerBehandlendeEnhet: KafkaProducer<String, KBehandlendeEnhetUpdate>,
+) {
+    fun updateBehandlendeEnhet(
+        pPerson: PPerson,
+    ) {
+        try {
+            val key = UUID.nameUUIDFromBytes(pPerson.personident.toByteArray()).toString()
+            kafkaProducerBehandlendeEnhet.send(
+                ProducerRecord(
+                    BEHANDLENDE_ENHET_UPDATE_TOPIC,
+                    key,
+                    KBehandlendeEnhetUpdate(
+                        pPerson.personident,
+                        pPerson.updatedAt,
+                    ),
+                )
+            ).get()
+            log.info("Record successfully sent to kafka, with key $key")
```

Comment:
Denne er for å teste at alt går fint for seg i dev, skal fjernes før prodsetting